paddle/fluid/framework/operator.cc

```diff
@@ -600,6 +600,23 @@ proto::VarType::Type OperatorWithKernel::IndicateDataType(
   return static_cast<proto::VarType::Type>(data_type);
 }
 
+bool OperatorWithKernel::CanCUDNNBeUsed(const ExecutionContext& ctx) const {
```

Comment:
I think it's better to make CanCUDNNBeUsed and CanMKLDNNBeUsed two global functions.

Reply:
@jacquesqiao where would you propose to place these functions?

Reply:
@pzelazko-intel according to this document https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/mkl/mkldnn_fluid.md#mkldnn_helper, we can put the interface there and add a `mkldnn_helper.cc`.

Reply:
done